10-setup-htcondor.sh

```diff
@@ -30,31 +34,49 @@ if [ "x$ANNEX_NAME" = "x" ]; then
     export ANNEX_NAME="$GLIDEIN_ResourceName@$GLIDEIN_Site"
 fi
 
-# token auth
-mkdir -p ~/.condor/tokens.d
-echo "$TOKEN" >~/.condor/tokens.d/flock.opensciencegrid.org
-chmod 600 ~/.condor/tokens.d/flock.opensciencegrid.org
-mkdir -p ~/.condor/passwords.d
-chmod 700 ~/.condor/passwords.d
+LOCAL_DIR=$(mktemp -d)
```

Comment:
```suggestion
LOCAL_DIR=$(mktemp -d osgvo-pilot-XXXXXXXXXX)
```

To retain the naming format of the old `LOCAL_DIR`

Reply:
If you want to use a template for mktemp:
1. if you don't give an absolute path it's interpreted as a relative path
2. only the last six X's are randomly filled in

So, `mktemp -d osgvo-pilot-XXXXXXXXXX` actually creates something like `osgvo-pilot-XXXXYP2kMl` in the current directory (rather than `/tmp`).

Reply:
```suggestion
LOCAL_DIR=$(mktemp -d /tmp/osgvo-pilot-XXXXXX)
```